samplers/samplers.go

```diff
@@ -182,13 +185,65 @@ func (c *Counter) Flush(interval time.Duration) []DDMetric {
 	copy(tags, c.Tags)
 	return []DDMetric{{
 		Name:       c.Name,
-		Value:      [1][2]float64{{float64(time.Now().Unix()), float64(c.value) / interval.Seconds()}},
+		Value:      [1][2]float64{{float64(time.Now().Unix()), float64(c.counts) / interval.Seconds()}},
 		Tags:       tags,
 		MetricType: "rate",
 		Interval:   int32(interval.Seconds()),
 	}}
 }
 
+// if this is a global veneur instance, flush the summed rates
+func (c *Counter) FlushGlobal(interval time.Duration) []DDMetric {
+	tags := make([]string, len(c.Tags))
+	copy(tags, c.Tags)
+	return []DDMetric{{
+		Name:       c.Name,
+		Value:      [1][2]float64{{float64(time.Now().Unix()), c.value}},
+		Tags:       tags,
+		MetricType: "rate",
+		// TODO (kiran): this is an okay approximation, as long ast
+		// the local and global instances are configured to flush at the
+		// same intervals. If this ever changes, this estimation will fail.
+		Interval: int32(interval.Seconds()),
```

Comment:
It's good for us to document this, though we're ~always guaranteed for the intervals to be the same (in fact, we're always guaranteed that the interval will be 10s, since changing that would require reconfiguring all of our metrics).